RocketCast/PlayerView.swift

```diff
@@ -22,6 +22,9 @@ class PlayerView: UIView {
         viewDelegate?.pausePodcast()
     }
     
+    @IBAction func changeSpeed(sender: UIButton) {
```

Comment:
i figured we would leave UI til the end, just focussing on getting it to work for now